# Setpoint for the UAV

float32 x       # x position
float32 y       # y position
float32 z       # z position

float32 roll    # roll
float32 pitch   # pitch
float32 yaw     # yaw